docs/docs/unreal/part-1.md

```diff
@@ -0,0 +1,132 @@
+# Unreal Tutorial - Part 1 - Setup
+
+Need help with the tutorial? [Join our Discord server](https://discord.gg/spacetimedb)!
+
+> A completed version of the game we'll create in this tutorial is available at:
+>
+> https://github.com/clockworklabs/SpacetimeDB/tree/master/demo/Blackholio
+
+## Prepare Project Structure
+
+This project is separated into two subdirectories;
+
+1. Server (module) code
+2. Client code
+
+First, we'll create a project root directory (you can choose the name):
+
+```bash
+mkdir blackholio
+cd blackholio
+```
+
+We'll start by populating the client directory.
+
+## Setting up the Tutorial Unreal Project
+
+In this section, we will guide you through the process of setting up a Unreal Project that will serve as the starting point for our tutorial. By the end of this section, you will have a basic Unreal project and be ready to implement the server functionality.
+
+### Step 1: Create a Blank Unreal Project
+
+SpacetimeDB supports Unreal version `5.6` or later. See [the overview](.) for more information on specific supported versions.
+
+Launch Unreal 5.6 and create a new project by selecting Games from the Unreal Project Browser.
+
+**⚠️ Important: Select the Blank** template and in **Project Defaults select C++**. 
+
+For `Project Name` use `client_unreal`. For Project Location make sure that you use your `blackholio` directory. This is the directory that we created in a previous step.
+
+Click "Create" to generate the blank project.
+
+![Create Blank Project](./part-1-01-create-project.png)
+
+### Import the SpacetimeDB Unreal SDK
+
+We're working on hosting the SpacetimeDB SDK for Unreal, in the mean time you can download from GitHub:
+> https://github.com/clockworklabs/SpacetimeDB/tree/master/sdks/unreal/src
```

Comment:
```suggestion
While the SpacetimeDB Unreal client SDK is in preview releases, it can only be installed from GitHub:

> https://github.com/clockworklabs/SpacetimeDB/tree/master/sdks/unreal/src

Once the SDK is stabilized, we'll find a more ergonomic way to distribute it.
```

Should also replace the link with one that actually goes directly to a download. I assume this will involve creating a GitHub release, but I'll leave the specifics to you and the Release Ops team.